### PR TITLE
Fix default string/text case sensitivity in Migrator

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

File: src/Model.php [hunk 1]
```diff
@@ -1676,9 +1676,11 @@ public function insert(array $row)
      */
     public function import(array $rows)
     {
-        foreach ($rows as $row) {
-            $this->insert($row);
-        }
+        $this->atomic(function () use ($rows) {
```
Comment (mvorisek) [inline src/Model.php]: now import is now only safer, but much faster ;-)

File: src-schema/Migration.php [hunk 2]
```diff
@@ -167,10 +168,16 @@ public function field(string $fieldName, $options = []): self
...
         }
 
-        if ($column->getType() === 'integer' && $refType !== self::REF_TYPE_NONE) {
+        if ($column->getType()->getName() === 'integer' && $refType !== self::REF_TYPE_NONE) {
             $column->setUnsigned(true);
         }
 
+        if (in_array($column->getType()->getName(), ['string', 'text'], true)) {
+            if ($this->getDatabasePlatform() instanceof SqlitePlatform) {
+                $column->setPlatformOption('collation', 'NOCASE');
```
Comment (mvorisek) [inline src-schema/Migration.php:177]: standard behaviour of MySQL string/text column (but only ASCII support)

**Comment by mvorisek** [comment]
test tested 🚀

File: src/Model.php (hunk 1 above)
Comment (DarkSide666) [reply src/Model.php]: I wonder why didn't we fix this ages ago. Multiple inserts in single transaction should give noticeable performance improvement.

**Review by DarkSide666 — APPROVED** [review]

File: src/Model.php (hunk 1 above)
Comment (mvorisek) [reply src/Model.php]: they were not in single transaction, but for each import a new transaction was started :D

Comment (mvorisek) [reply src/Model.php]: as major, moved to saparate PR - https://github.com/atk4/data/pull/767

**Pull request merged**